core/alarms-lambda.js

```diff
@@ -23,7 +23,7 @@ module.exports = function LambdaAlarms (functionAlarmConfigs, context, serverles
     )
 
     for (const [funcResourceName, funcResource] of Object.entries(lambdaResources)) {
-      const functionName = funcResource.Properties.FunctionName
+      const functionName = funcResource.Properties.FunctionName ? funcResource.Properties.FunctionName : `${funcResourceName}Name`
```

Comment:
I've updated this but the alarm named looks weird like 'LambdaThrottles_[object Object]', same for API alarms. have to figure out how to solve that issue.

Reply:
If `{ Ref: funcResourceName }` gets stringified in JavaScript at some point, it will result in `[object Object]`. In that case, we canuse a CloudFormation `Sub` to interpolate the function name instead. So the alarm name could be set as `!Sub "LambdaThrottles_${funcResourceName}"`. That will let CloudFormation expand the variable name instead of JavaScript.

Reply:
Okay, thanks. Trying that now.